src/util/irep.h

```diff
@@ -29,6 +29,8 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <list>
 #else
 #include <map>
+#include <unordered_set>
```

Comment:
This is in the wrong place for it would not get included when `SUB_IS_LIST` is set.